.gitignore

```diff
@@ -57,3 +57,6 @@ README.html
 
 .ipynb_checkpoints/
 *.ipynb
+
+#pycharm
+.idea
```

Comment:
I'm -0 on adding things like this to project gitignores. This should really be in a global gitignore. See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore